internal/scyllaCloudSDK/endpoints.go

```diff
@@ -0,0 +1,79 @@
+package scyllaCloudSDK
```

Comment:
please don't name package like this, no camel-case

just `scylla` is enough, especially when provider is named terraform-provider-`scylla`